src/components/WorkshopPage/Workshop.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import ReactPlayer from 'react-player/youtube';
+import Container from '@material-ui/core/Container';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useTheme from '@material-ui/core/styles/useTheme';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+
+const workshops = [
+	{
+		type: 'Web Development',
+		elements: [
+			{
+				title: 'Intro to HTML',
+				author: 'Eric',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Intro to CSS',
+				author: 'Chandra',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Intro to HTML',
+				author: 'Rick Astley',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			}
+		]
+	},
+	{
+		type: 'Mobile Development',
+		elements: [
+			{
+				title: 'Reactive Native',
+				author: 'Jakob',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			},
+			{
+				title: 'Android Studio',
+				author: 'Nareh',
+				description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit,',
+				url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO'
+			}
+		]
+	}
+];
+
+function Workshop() {
+	const theme = useTheme();
+	const isSmall = useMediaQuery(theme.breakpoints.down('sm'));
+
+	const workshopCards = workshops.map(item =>
+		<Grid key={item.type} style={{ paddingBottom: theme.spacing(isSmall ? 4 : 8) }}>
+			<Typography style={{ fontWeight: theme.typography.fontWeightMedium,
+				paddingBottom: theme.spacing(isSmall ? 4 : 8) }}
+			variant='h3' component='h2'>{item.type}
+			</Typography>
```

Comment:
Let's clean this up a bit stylistically.

```suggestion
			<Typography
				style={{
					fontWeight: theme.typography.fontWeightMedium,
					paddingBottom: theme.spacing(isSmall ? 4 : 8)
				}}
			  variant='h3'
				component='h2'
			>
				{item.type}
			</Typography>

```